web/src/app/administration/access-items-management/access-items-management.component.ts

```diff
@@ -53,6 +54,27 @@ export class AccessItemsManagementComponent implements OnInit, OnDestroy {
   custom11Field = this.customFieldsService.getCustomField('Custom 11', 'workbaskets.access-items.custom11');
   custom12Field = this.customFieldsService.getCustomField('Custom 12', 'workbaskets.access-items.custom12');
 
+  constructor(private formBuilder: FormBuilder,
+    private customFieldsService: CustomFieldsService,
+    private accessIdsService: AccessIdsService,
+    private formsValidatorService: FormsValidatorService,
+    private requestInProgressService: RequestInProgressService,
+    private removeConfirmationService: RemoveConfirmationService,
+    private alertService: AlertService,
+    private generalModalService: GeneralModalService,
```

Comment:
I think generalModalService is not in use anymore. Please verify and delete together with import.